orangecontrib/prototypes/widgets/owsplit.py

```diff
@@ -0,0 +1,136 @@
+import Orange.data
+import numpy as np
+from Orange.data import Table
+from Orange.data.util import SharedComputeValue, get_unique_names
+from Orange.widgets import gui
+from Orange.widgets.settings import ContextSetting, DomainContextHandler
+from Orange.widgets.widget import OWWidget, Msg, Output, Input
+from AnyQt.QtCore import Qt
+
+from Orange.data import DiscreteVariable, StringVariable
+from Orange.widgets.utils.itemmodels import DomainModel
+from Orange.widgets.utils.widgetpreview import WidgetPreview
+from orangewidget.settings import Setting
+
+
+def get_string_values(data, var):
+    # turn discrete to string variable
+    column = data.get_column_view(var)[0]
+    if var.is_discrete:
+        return [var.str_val(x) for x in column]
+    return column
+
+
+class SplitColumn:
+    def __init__(self, data, attr, splitter):
```

Comment:
When I first read this, I thought `splitter` would be a function that splits the string.

Perhaps this should be called `delimiter`, not `splitter`.

Same in widget's code.